frontend/src/App.tsx

```diff
@@ -17,7 +17,9 @@ import ChangePassword from "./pages/ChangePassword";
 import ChargingTable from "./pages/Charging/ChargingTable";
 import { ProtectedRoute } from "./ProtectedRoute";
 import { LoginContext, User } from "./LoginContext";
-
+import ProfileList from "./pages/ProfileList";
+import ProfileCreate from "./pages/ProfileCreate";
+import ProfileRead from "./pages/ProfileRead";
 export default function App() {
```

Comment:
```suggestion
import ProfileRead from "./pages/ProfileRead";


export default function App() {
```

Add blank lines between imports and function declarations.